app/libs/utils/date.ts

```diff
@@ -31,6 +31,15 @@ export const extractTimeFromDate = (date: string | undefined | null) => {
   return `${hours}:${minutes}`
 }
 
+/**
+ * @param date  - ISO date string
+ * @returns - Formatted date string e.g. '2023-01-01'
+ */
+export const formatDbDate = (date: string | undefined | null) => {
+  const formattedDate = date?.split('T')[0]
+  return formattedDate
+}
+
```

Comment:
You don't use this function? If so, could you delete it?

Reply:
Sorry about that. I noticed that formatDbDate is not being used. I removed the function.